.env.example

```
@@ -24,3 +24,6 @@ NEXT_PUBLIC_RPC_SCROLL=
 NEXT_PUBLIC_E2E_AUTH_BYPASS=false
 
 NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=
+
+NEXT_PUBLIC_LANGFUSE_PUBLIC_KEY=
+NEXT_PUBLIC_LANGFUSE_HOST=https://cloud.langfuse.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_LANGFUSE_HOST key should go before the NEXT_PUBLIC_LANGFUSE_PUBLIC_KEY key

(UnorderedKey)